unitex/src/fr/umlv/unitex/leximir/delas/ChooseDelas.java

```diff
@@ -177,8 +177,8 @@ public boolean accept(File dir, String name) {
                 int reply = JOptionPane.showConfirmDialog(null, "No Delas folder found in Dela folder \nCreate Delas folder?", title,
                         JOptionPane.YES_NO_OPTION);
                 if (reply == JOptionPane.YES_OPTION) {
-                    allDFoler.mkdirs();
-                    if (allDFoler.exists()) {
+                    allDFolder.mkdirs();
+                    if (allDFolder.exists()) {
                         JOptionPane.showMessageDialog(null, "The folder created successfully !", "Information", JOptionPane.INFORMATION_MESSAGE);
```

Comment:
What about `The folder has been successfully created`

Notice that it is not necessary to add a non-breaking space before the exclamation mark.

Reply:
Done!